daemon/graphdriver/vfs/driver.go

```diff
@@ -16,8 +15,8 @@ import (
 )
 
 var (
-	// CopyWithTar defines the copy method to use.
-	CopyWithTar = chrootarchive.NewArchiver(nil).CopyWithTar
+	// CopyDir defines the copy method to use.
+	CopyDir = dirCopy
```

Comment:
No need for this one, the functions can call `copyDir` directly.

Reply:
Oh nevermind I see the layerstore test is using this... something about nice things...

Reply:
👍

Reply:
@cpuguy83 Anything else then?

Reply:
Is `chrootarchive` really necessary now, without it then there would be no need to expose this, especially since Linux no longer needs it...

Reply:
What would we do with the non-windows bits?